spyql/utils.py

```diff
@@ -27,8 +28,30 @@ def try2eval(val, globals={}, locals={}):
 
 
 def isiterable(x):
-    """Returns True if `x` is iterable that is not a string and is not null"""
+    """Returns True if `x` is iterable that is not a string or dict and is not null"""
     from spyql.nulltype import Null
     from collections.abc import Iterable
 
-    return isinstance(x, Iterable) and x is not Null and not isinstance(x, str)
+    return (
+        isinstance(x, Iterable)
+        and x is not Null
+        and not isinstance(x, str)
+        and not isinstance(x, dict)
+    )
+
+
+def is_row_collapsable(row, colnames):
+    """
+    Returns True if `row` is only has a single column with a default name and is a dict.
```

Comment:
🔧 I feel there's a typo here, but not sure what was your intended meaning. The english feels off.
```suggestion
    Returns True if `row` only has a single column with a default name and is a dict.
```

Reply:
Will change to:

```
Returns True if `row` only has a single column of type dict with a default name.
```

Reply:
https://github.com/dcmoura/spyql/pull/68/commits/5e7e73bb43ee738e9ca1d334c3dc1cfb73688c98